data/dev/gigadb_user.csv

```diff
@@ -1,6 +1,7 @@
 id,email,password,first_name,last_name,affiliation,role,is_activated,newsletter,previous_newsletter_state,facebook_id,twitter_id,linkedin_id,google_id,username,orcid_id,preferred_link
 8,test+8@gigasciencejournal.com,5a4f75053077a32e681f81daa8792f95,Guojie,Zhang,BGI,"",f,t,t,,,,,test+8@gigasciencejournal.com,,EBI
 14,test+14@gigasciencejournal.com,5a4f75053077a32e681f81daa8792f95,Shifeng,Cheng,BGI,"",f,t,t,,,,,test+14@gigasciencejournal.com,,EBI
+38,admin1@gigasciencejournal.com,bcf839ae349d13cb0c3c4a9ec0cc243e,C,H,BGI,admin,t,t,t,,,,114715812788877222335,admin1@gigasciencejournal.com,,EBI
```

Comment:
@rija This line seems to be causing the acceptance test for the `Users` scenario in `AdminTablesFilter.feature` to fail for me when executed locally and on CI/CD pipeline. This scenario fills in the search text field for the `Email` column with "admin" which detects this `admin1` user but the `And I should not see "@gigasciencejournal.com"` scenario step then cannot be passed.